docs/installation/source.rst

```diff
@@ -37,36 +37,40 @@ please follow the directions :ref:`here <contributing>`.
        On Fedora Linux, you must replace ``pip`` with ``pip-python`` in the
        following steps.
 
-#. Then you'll need to install GStreamer 0.10 (>= 0.10.31, < 0.11), with Python
-   bindings. GStreamer is packaged for most popular Linux distributions. Search
-   for GStreamer in your package manager, and make sure to install the Python
+#. Then you'll need to install GStreamer 1.x (>= 1.2), with Python bindings.
```

Comment:
Fixed in 1daf5825580d31e3f2825b5b5edfaa2aed8146fe using same error style as the Python version check in `mopidy/__init__.py`. I didn't want to have the GStreamer version check there, as I want as large parts of Mopidy as possible to be importable without GStreamer to make it possible for some extensions to be tested with just Mopidy in a virtualenv without site-packages.